src/modules/0.0.22/aws_s3/mappers/bucket.ts

```diff
@@ -135,7 +129,8 @@ export class BucketMapper extends MapperBase<Bucket> {
         const { bucketName, region } = this.idFields(id);
```

Comment:
this won't be working since the entity property with the cloud id is `name` and not `bucketName`